packages/nextjs-kit/src/lib/sortDate.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param {Array<Record<string, string | number>>} obj object to sort
+ * @param {string} key to sort object by
+ * @param {string} direction to sort object by
+ * @returns {Array<Record<string, string | number>>} the sorted object that was passed in
+ * @remarks
+ * This is a function to sort any date filed of an object on a specific key in a direction of the users choice.
+ *
+ * Note:
+ * There is a named and default export so that we can
+ * support submodules for the ESModule build and a single entrypoint for the UMD build
+ */
+export const sortDate = (
+  obj: Array<Record<string, string>>,
+  key: string,
+  direction: string
+) => {
```

Comment:
Also make these params an object. Maybe create an interface that can be shared between these helpers?